### PR TITLE
LFA-1479 Show correct text when only confirmation statement is overdue

### PR DESCRIPTION
**Pull request by mwestacott**

### CODE REVIEW

File: src/controllers/confirmation.controller.ts [hunk 1]
```diff
@@ -119,4 +115,18 @@ const route = async (req: Request, res: Response, next: NextFunction): Promise<v
   }
 };
 
+const getOverdueFiling = (companyProfile: PTFCompanyProfile): string => {
+  let overdueFiling: string = "";
+
+  if (companyProfile.isAccountsOverdue && !companyProfile.isConfirmationStatementOverdue) {
```
Comment (mwestacott) [inline src/controllers/confirmation.controller.ts:121]: I think this is the most readable format. @markpit what do you think?

Comment (markpit) [reply src/controllers/confirmation.controller.ts:121]: looks good

Comment (markpit) [reply src/controllers/confirmation.controller.ts:121]: there is actually a trick I discovered.
const getOverdueFiling = ({isAccountsOverdue, isConfirmationStatementOverdue}): string =>

call the method as normal, you can then use the fields from companyProfile directly!

Comment (mwestacott) [reply src/controllers/confirmation.controller.ts]: Yes, that seems to work a treat. Updated.

**Pull request merged**